data-raw/manifest.R

```diff
@@ -1,11 +1,40 @@
 library(data.table)
 
-# -------------------------------- EPIC chip manifest
+# -------------------------------- EPIC V2 chip manifest
 
-### CSV contains both 'normal' and control probes. Create two separate tables for them (split at line 865927)
+### NEW EPIC array added by Costanza L. Vallerga
+### CSV contains both 'normal' and control probes. Create two separate tables for them (split at line 937055)
+
+manifest_epic_v2 = fread("EPIC-8v2-0_A1.csv",skip="IlmnID",header=TRUE,nrows=937055,integer64="character",sep=",",sep2=";")
+
+manifest_epic_v2 = manifest_epic_v2[,list(
+     probe_id=IlmnID
```

Comment:
```suggestion
     probe_id=Name
```

The manifest columns changed a bit from v1. `IlmnID` is no longer just the probe name.